### ISSUE TITLE
Feedback for: compose/wordpress.md

### ISSUE THREAD

**Issue by still-dreaming-1** [docker/docker.github.io#129]
URL: [https://docs.docker.com/compose/wordpress.md](https://docs.docker.com/compose/wordpress/)

I got to the `docker-compose up -d` part, which gives me the error:

> ERROR: yaml.parser.ParserError: while parsing a block mapping
>   in "./docker-compose.yml", line 1, column 1
> expected <block end>, but found '<block mapping start>'
>   in "./docker-compose.yml", line 2, column 2

I'm on Linux Mint 17.3 64 bit Cinnamon edition. Which is based on Ubuntu 14.04. I started out with no docker anything installed. I have Docker version 1.12.1, build 23cf638. My docker composer version is 1.8.1.

**Comment by londoncalling** [comment]
@still-dreaming-1 @dnephin 

Thank you @still-dreaming-1 , I believe the problem is caused by indentation errors (similar to this issue: https://forums.docker.com/t/errors-on-docker-compose-up-parser-error/14747/3). I will update the docs to reflect the right spacing. 

Try updating the first few lines of `docker-compose.yml` to look like this instead (i.e., add a return after the version line, leave all other indentation and spacing the same). Then your `docker-compose up -d` should work:

```
version: '2'

 services:
   db:
     image: mysql:5.7
     volumes:
       - "./.data/db:/var/lib/mysql"
. . .
```

**Comment by dnephin** [comment]
It looks like the example has an extra space as the first character on each line. `version` and `services` should be at the same indentation level.

**Comment by londoncalling** [comment]
Thanks, @dnephin when I made the correction I provided above, it worked, but I'll also make the corrections you suggest, re-test, and modify the example to reflect the right spacing and indentation.

**Comment by londoncalling** [comment]
This is fixed now in the docs per PR #359
